core/src/test/java/org/apache/iceberg/rest/TestHTTPClient.java

```diff
@@ -62,6 +51,17 @@
 import org.mockserver.model.HttpResponse;
 import org.mockserver.verify.VerificationTimes;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
you might need to double-check your IDE settings, but static imports need to be at the top